src/test/java/org/jabref/logic/bibtexkeypattern/BibtexKeyGeneratorTest.java

```diff
@@ -239,7 +239,7 @@ void testMakeLabelAndCheckLegalKeysAccentGrave() throws ParseException {
 
         entry0 = BibtexParser.singleFromString("@ARTICLE{kohn, author={Andrés Aʹrnold}, year={2000}}",
                 importFormatPreferences, fileMonitor);
-        assertEquals("Arn",
```

Comment:
This change is strange. `Arn` read better than `A´r`. Unsure, what the other reviewers think here. -- On the one hand, this is an example that keeping as much characters as possible might not be good. On the other hand, we have the cleanup funtionality for BibTeX keys, which can be used by users.

Reply:
I think in this case should we stay with `Arn`, but when we removed `ʹ` from `KEY_ILLEGAL_CHARACTERS` this test was going fail. I wasn't sure what to do with that so that`s why i change to `A´r`.